src/pages/style.css

```diff
@@ -162,7 +162,7 @@ and negative margin.
 
   &:focus-visible {
     border: 1px solid var(--token-color-palette-blue-200);
```

Comment:
@ashleemboyer can you double check if `blue-200` is the right color (maybe is a specific design decision)? for us in HDS this is `blue-300` which means this can be declared using the specific token `var(--token-color-focus-action-internal)`

Reply:
@didoo I assume this is something that just hasn't been updated in a while, or since the focus tokens were added. I say we leave this one for now because the `.g-focus-border-and-box-shadow-light` class has a wide area of impact and there are plans to refactor how focus styles are handled for many components.

Reply:
OK for me. Maybe we can add a comment, in this case? Like a TODO? What do you think?

Reply:
I think the TODO above `.g-focus-ring-from-box-shadow` will suffice. 😊